### PR TITLE
Starting to make the base UI

### PR DESCRIPTION
**Pull request by mneedleman**
image

We can work on spacing later.

### CODE REVIEW

**Review by rao53 — COMMENTED** [review]
I believe you should separate the Header and Footer into separate components. It would make it easier to use across various pages

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -13,16 +36,69 @@ function MainPage() {
...
+                className={classes.header}
+            >
+                <Typography variant='h5' className={classes.bannerText}>
+                    {'Hi, Jane'}
+                </Typography>
+                {/* Figure out sizes */}
+                <Grid item lg={2} sm={2} xs={2}>
+                    <Grid container direction='row' justify='space-between'>
+                        <Typography className={classes.bannerText}>
+                            {'About'}
```
Comment (jcarrete5) [inline frontend/src/App.tsx:58]: Why is text embedded as javascript string literals?

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -13,16 +36,69 @@ function MainPage() {
...
+                direction='row'
+                justify='space-between'
+                alignItems='center'
+                className={classes.header}
+            >
+                <Typography variant='h5' className={classes.bannerText}>
+                    {'Hi, Jane'}
+                </Typography>
+                {/* Figure out sizes */}
+                <Grid item lg={2} sm={2} xs={2}>
```
Comment (jcarrete5) [inline frontend/src/App.tsx:55]: I think you can get away with only specifying the smallest breakpoint (xs) here and the larger values will default to that value unless specified but I'm having trouble finding the reference.

**Comment by jcarrete5** [comment]
> I believe you should separate the Header and Footer into separate components. It would make it easier to use across various pages

The header and footer already do this because each page will be a route specified in the switch element and thus will have the header and footer above and below.

File: frontend/src/App.tsx (hunk 2 above)
Comment (mneedleman) [reply frontend/src/App.tsx:55]: That's a good point. I'll double check that

File: frontend/src/App.tsx (hunk 1 above)
Comment (mneedleman) [reply frontend/src/App.tsx:58]: I just prefer it that way

Comment (mneedleman) [reply frontend/src/App.tsx:58]: I just prefer it that way

**Review by jcarrete5 — APPROVED** [review]

**Comment by jcarrete5** [comment]
This can be merged, what are we waiting for? The code has been reviewed by every reviewer and approved by at least 1.

**Review by jcarrete5 — APPROVED** [review]

**Pull request merged**